crates/apollo-mcp-registry/src/uplink/schema.rs

```diff
@@ -181,12 +181,6 @@ impl SchemaSource {
     }
 }
 
-#[derive(thiserror::Error, Debug)]
-enum FetcherError {
-    #[error("failed to build http client")]
-    InitializationError(#[from] reqwest::Error),
-}
```

Comment:
Why is this removed? Was it unused?

Reply:
Yeah, I'm not sure why but it was unused this whole time?